experimental/dds/tree2/src/test/shared-tree/opSize.bench.ts

```diff
@@ -529,13 +529,48 @@ describe("Op Size", () => {
 	});
 
 	describe("Insert, Delete & Edit Nodes", () => {
+		const oneThirdNodeCount = Math.floor(BENCHMARK_NODE_COUNT * (1 / 3));
+		const seventyPercentCount = BENCHMARK_NODE_COUNT * 0.7;
```

Comment:
While you're at it, can you `Math.floor` this count and the one below too? It shouldn't change the current behavior, but it will be nice to have all the "counts" be integers if anyone is ever adjusting `BENCHMARK_NODE_COUNT`. Technically, they should probably all be `Math.ceiling`, since a benchmark should error on the conservative side, but that's a separate concern.

Reply:
Can do.